### ISSUE TITLE
Map and Set aren't iterable when native Map and Set aren't present

### ISSUE THREAD

**Issue by AgentME** [zloirock/core-js#73]
```
$ node
> var CSymbol = require('core-js/library/fn/symbol');
undefined
> var CMap = require('core-js/library/fn/map');
undefined
> var m = new CMap();
undefined
> m[CSymbol.iterator];
undefined
> m.values()[CSymbol.iterator];
[Function]
```

The situation that this is presenting a problem to me is that we're using [safari-fix-map](https://www.npmjs.com/package/safari-fix-map) which deletes Safari's native Map and Set objects and replaces them with core-js's versions (We've had ridiculous issues with native Maps in Safari where they're not consistent in some situations), and then we have (babel-transpiled with "runtime" transformation) code which creates some sets, tries to for-of loop over them, and fails because the Symbol.iterator property is missing from the sets. Using a for-of loop over the return value of certain methods like `values` works though.

**Comment by zloirock** [comment]
`[Symbol.iterator]` property not adds in library version `core-js` for prevent pollute collections prototypes when exist native collections. For get iterator in library version you shoud use `core.getIterator`. For your `safari-fix-map` more convenient non-library version `core-js`.

**Comment by AgentME** [comment]
Oh, so apparently the sets can still be iterable without a `[Symbol.iterator]` property. Babel already uses `core.getIterator`.

Though there still is some bug actually causing exceptions to happen when we try to iterate with a for-of loop over sets in Safari, though I haven't figured out the exact conditions that lead to this yet. When I find a test case that shows the issue I'll open a new bug.

**Comment by zloirock** [comment]
@AgentME looks like you're right, I see possible conflict if will be loaded some different `library` instances w/o global polyfill. I'll try to fix it in the next days.

**Comment by zloirock** [comment]
Should be fixed in [`0.9.15`](https://github.com/zloirock/core-js/releases/tag/v0.9.15).
